### PR TITLE
test : Refactor hard-coded status codes to HTTPstatus in tests

### LINKED ISSUE #968: Use HTTPStatus instead of hardcoded values in tests/task_comments
## Description
As a developer,
I need to avoid using hard-coded values wherever possible,
so that I can make code more readable and scalable.

### Update [Required]
- [ ] Use HTTPStatus instead of hard-coded response status codes (like in the API) in `tests/task_comments`

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by cypherean**
### Description

Refactored `tests/task_comments/` and changed hard-coded status codes to respective HTTPstatus.

Fixes #968 

### How Has This Been Tested?

Screenshot from 2021-01-21 19-37-32

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tests/task_comments/test_api_create_task_comment.py [hunk 1]
```diff
@@ -40,7 +41,7 @@ def test_task_comment_creation_api_with_comment_not_string(self):
             data=json.dumps(dict(comment=5)),
         )
 
-        self.assertEqual(400, actual_response.status_code)
+        self.assertEqual(HTTPStatus.BAD_REQUEST, actual_response.status_code)
```
Comment (vj-codes) [inline tests/task_comments/test_api_create_task_comment.py]:
```suggestion
        self.assertEqual(HTTPStatus.BAD_REQUEST.value, actual_response.status_code)
```

File: tests/task_comments/test_api_create_task_comment.py [hunk 2]
```diff
@@ -26,7 +27,7 @@ def test_task_comment_creation_api_without_comment(self):
             data=json.dumps(dict(example="example")),
         )
 
-        self.assertEqual(400, actual_response.status_code)
+        self.assertEqual(HTTPStatus.BAD_REQUEST, actual_response.status_code)
```
Comment (vj-codes) [inline tests/task_comments/test_api_create_task_comment.py]:
```suggestion
        self.assertEqual(HTTPStatus.BAD_REQUEST.value, actual_response.status_code)
```

**Review by vj-codes — CHANGES_REQUESTED** [review]
@cypherean hey please change the code as mentioned in the suggested changes in every file of the PR . Just these small changes and you are done:)

**Comment by cypherean** [comment]
Made the required changes @vj-codes :smile:

**Comment by vj-codes** [comment]
@cypherean hey great work on updating the changes!
Although I still see some places where` .value` isn't added , this will not produce the expected output so please add it in those places as well.

**Review by Aaishpra — CHANGES_REQUESTED** [review]
Hi @cypherean can you run black tests/folder_name_here/file_name_here.py for all the files or black . so that the code will be reformatted according to black tool standards and the failing check will pass.

**Review by vj-codes — APPROVED** [review]
LGTM 🎉 
Thank you for your contribution @cypherean

**Comment by vj-codes** [comment]
@Aaishpra hey can you review this once more?

**Review by Aaishpra — APPROVED** [review]
Thanks for the quick change @cypherean great work.

**Review by chinmaym07 — APPROVED** [review]
LGTM :+1:
Thanks @cypherean for your contribution

Tested locally and verified that all hard coded response values are replaced

The changes made in this PR were tested locally. Following are the results:

1) Code review - Done

2) All possible responses (positive and negative tests) were tested as below:

     - Test1 Description Replaced hard coded responses in mentioned files
    - Expected Result: tests/task_comments files should not contain hard coded responses

 3) Additional Comments: Run `python -m unittest discover tests to ensure all tests passed

**Comment by epicadk** [comment]
> LGTM 👍
> Thanks @cypherean for your contribution
> 
> Tested locally and verified that all hard coded response values are replaced
> 
> The changes made in this PR were tested locally. Following are the results:
> 
> 1. Code review - Done
> 2. All possible responses (positive and negative tests) were tested as below:
>    
>    * Test1 Description Replaced hard coded responses in mentioned files
>    * Expected Result: tests/task_comments files should not contain hard coded responses
> 3. Additional Comments: Run `python -m unittest discover tests to ensure all tests passed

Can you add a screenshot of the tests?

**Comment by isabelcosta** [comment]
@epicadk thank you for asking for more clarifications 🤗 I merged, since the PR check was passing and these were changes made to the tests module only.
Thank you all for reviewing this 👏

**Pull request merged**